### ISSUE TITLE
Spark Alliance Paypal account

### ISSUE THREAD

**Issue by aladar42** [hongkonggong/beyond-lennon-walls#17]
Hello, as far as I know, Spark got their Paypal account closed about two weeks back due to not being a "proper" charity with paperwork. At least that was the case last week, confirmed by the Spark staff themselves on Facebook. Could someone clarify and update this?

**Comment by hongkonggong** [comment]
Yeah, it seems not to work right now. (Verified through trying to send money through PayPal and through their FB post.) Amending the document shortly.
